db/schema.rb

```diff
@@ -103,15 +106,12 @@
     t.integer "student_loan_plan"
     t.string "eligibility_type"
     t.uuid "eligibility_id"
-    t.index ["claim_school_id"], name: "index_tslr_claims_on_claim_school_id"
-    t.index ["current_school_id"], name: "index_tslr_claims_on_current_school_id"
     t.index ["eligibility_type", "eligibility_id"], name: "index_tslr_claims_on_eligibility_type_and_eligibility_id"
-    t.index ["employment_status"], name: "index_tslr_claims_on_employment_status"
```

Comment:
Did we intentionally lose this index? I think it makes sense, but just wanted to check.

Reply:
Interesting, had no idea the original column had an index. I don't think we actually need it on the new table, it certainly doesn't feel like an attribute we're likely to query against any time soon.

Reply:
lol, looks like myself and @mec added the original index..